packages/dds/tree/src/test/sequence-change-family/testGeneratedChanges.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { strict as assert } from "assert";
+import * as fs from "fs";
+import { toDelta, Transposed as T } from "../../changeset";
+import { sequenceChangeRebaser } from "../../feature-libraries";
+import { verifyChangeRebaser } from "../../rebase";
+import { FieldKey } from "../../tree";
+import { brand } from "../../util";
+import { generateFuzzyCombinedChange } from "../rebase/fuzz";
+import { generateRandomChange, generateRandomUpPath } from "./randomSequenceGenerator";
+
+const fooKey = brand<FieldKey>("foo");
+const keySet = new Set([fooKey]);
+
+const pathGen = (seed: number) => generateRandomUpPath(keySet, seed, 3, 5);
+describe("testing the SequenceChangeRebaser", () => {
+    it("using generateRandomChange function", async () => {
+        const changes = new Set<T.LocalChangeset>();
+        for (let i = 0; i <= 15; i++) {
+            const change = generateRandomChange(Math.floor(Math.random() * 100), pathGen);
```

Comment:
We want the use a seeded random function to ensure the tests are deterministic.
Use `const random = makeRandom(seed);` with a constant seed.